articles/commerce/pricing-apis.md

```diff
@@ -527,4 +550,190 @@ The following table shows the input parameters for the *RemoveCoupons* API.
 | key         | string | Required | The cart ID. |
 | couponCodes | IEnumerable\<string\> | Required | The coupon codes to remove from the cart. |
 
+## GetProductPromotions
+
+The *GetProductPromotions* API was introduced in the Commerce version 10.0.38 release. 
+
+This API gets a list of promotional products with given product discounts.
```

Comment:
Is this "1 sentence 1 paragraph" the pattern? It makes the reading more difficult.